incubator/java-openliberty/stack.yaml

```diff
@@ -19,3 +19,5 @@ maintainers:
 default-template: default
 requirements:
     docker-version: ">= 17.09.0"
+templating-data:
+     liberty_version: '19.0.0.12'
```

Comment:
The README.md references this version.. would it make sense to have some instruction to a stack author customizing our stack somewhere?

Reply:
I marked PR "approved" anyway.. just a doc thought.